packages/admin-app/src/components/ElectionSetupWizard/Steps/BasicInfoStep.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Container, IconButton, TextField, Tooltip } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import React, { useState } from 'react';
+import InfoIcon from '@mui/icons-material/Info';
+
+import { useIntl } from 'react-intl';
+import { SubmitElectionRequest } from '@electionguard/api-client';
+import FormattedButton from '../../FormattedButton';
+import IconHeader from '../../IconHeader';
+import { Message, MessageId } from '../../../lang';
+
+export interface SetupInstructionsStepProps {
+    onNext?: (submitElectionRequest: SubmitElectionRequest) => void;
+}
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        padding: 0,
```

Comment:
This padding should also be unnecessary. We should be relying on padding of the container.